.github/workflows/rust.yml

```diff
@@ -24,8 +24,8 @@ jobs:
     - name: Run cargo check without any default features
       uses: actions-rs/cargo@v1
       with:
-        command: check
-        args: --no-default-features
+        command: build
+        args: --locked --release --no-default-features
```

Comment:
you can remove it altogether, meilisearch is built by the test command.

Reply:
it will Make CI even slower, since the build release was used by the test thanks to incremental compilation, now it just adds an extra step :)

Reply:
It was to test if the build with `--no-default-features` works since it's possible the default build work but the `--no-default-features` don't. That's why I added this step

Reply:
my bad then, this is correct :)